packages/web3-eth/src/types.ts

```diff
@@ -32,6 +32,7 @@ export type SendTransactionEvents<ReturnFormat extends DataFormat> = {
 		latestBlockHash: FormatType<Bytes, ReturnFormat>;
 	};
 	error: TransactionError<FormatType<TransactionReceipt, ReturnFormat>>;
+	contractExecutionError: ContractExecutionError | ResponseError<any>;
```

Comment:
Considering high cohesive, if we need to add `contract` execution error in `SendTransactionEvents` type, then this type should be moved to web3-types package instead of keeping in eth package.